specs/listening_ports.table

```diff
@@ -2,10 +2,14 @@ table_name("listening_ports")
 description("Processes with listening (bound) network sockets/ports.")
 schema([
     Column("pid", INTEGER, "Process (or thread) ID"),
-    Column("port", INTEGER, "Transport layer port"),
+    Column("port", TEXT, "Transport layer port"),
```

Comment:
Why this change?

Reply:
I have changed this to allow for empty values when adding rows about AF_UNIX sockets

Reply:
Can't port be null which is an empty value?

Reply:
Hey @obelisk ! How can you pass a NULL value to a column using the Row type? The code in virtual_table.cpp will always attempt to parse the value as an integer